### PR TITLE
ENH: add sliders in sympy etapipi notebook

### LINKED ISSUE #94: Make plots interactive with widgets for `sympy` etapip notebook
Make plots interactive with widgets (mass, width, coefficients, and phase)

### PR DESCRIPTION
**Pull request by shenvitor**
- Closes #94

### CODE REVIEW

File: docs/eta-pi-p/manual.ipynb [hunk 1]
```diff
@@ -674,8 +888,8 @@
      "source_hidden": true
     },
     "tags": [
-     "hide-input",
-     "full-width"
+     "full-width",
+     "hide-input"
```
Comment (redeboer) [inline docs/eta-pi-p/manual.ipynb:892]: Make the plot with the histograms interactive as well. Can be done in the same PR now that you know the trick ;)

File: docs/eta-pi-p/manual.ipynb [hunk 2]
```diff
@@ -601,6 +672,98 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
```
Comment (redeboer) [inline docs/eta-pi-p/manual.ipynb:682]: This cell with slider definitions should be `hide-input` and `scroll-input`

File: docs/eta-pi-p/manual.ipynb [hunk 3]
```diff
@@ -601,6 +672,98 @@
...
+    "tab_contents = []\n",
+    "resonances_name = [\"N*\", \"Δ*\", \"a₂*\"]\n",
+    "for i in range(len(resonances_name)):\n",
+    "    tab_content = w.VBox([\n",
+    "        w.HBox(categorized_sliders_m[i] + categorized_sliders_gamma[i]),\n",
+    "        w.VBox(categorized_cphi_pair[i]),\n",
+    "    ])\n",
+    "    tab_contents.append(tab_content)\n",
+    "UI = w.Tab(tab_contents, titles=resonances_name)\n",
+    "UI"
```
Comment (redeboer) [inline docs/eta-pi-p/manual.ipynb:764]: Don't display the `UI` here
```suggestion
    "UI = w.Tab(tab_contents, titles=resonances_name)"
```

**Review by redeboer — CHANGES_REQUESTED** [review]

File: docs/eta-pi-p/manual.ipynb [hunk 4]
```diff
@@ -171,7 +184,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "s23, m_delta, gamma_delta = sp.symbols(\"s_{23} m_Delta Gamma_Delta\")\n",
+    "s23, m_delta, gamma_delta = sp.symbols(r\"s_{23} m_{\\Delta^+} \\Gamma_{\\Delta^+}\")\n",
```
[the remark this answers is not in the document]
Comment (shenvitor) [reply docs/eta-pi-p/manual.ipynb:187]: I don't agree..

[the remark this answers is not in the document]
Comment (redeboer) [reply docs/eta-pi-p/manual.ipynb:190]: See https://github.com/MagicStack/MagicPython/issues/114

**Review by redeboer — APPROVED** [review]
Cool 😎

**Pull request merged**